backend/indexer/indexer.py

```diff
@@ -82,54 +90,55 @@ async def sync_data_source_to_collection(inputs: DataIngestionConfig):
         )
     except Exception as e:
         logger.exception(e)
-        await client.aupdate_data_ingestion_run_status(
+        _run_async( client.aupdate_data_ingestion_run_status(
```

Comment:
We need not change await calls, they already run on the same event loop managed by uvicorn and starlette